phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -3165,10 +3184,12 @@ private MetaDataMutationResult mutateColumn(
                 if (mutationResult.isPresent()) {
                     return mutationResult.get();
                 }
-
+                // We take a write row lock for tenantId, schemaName, tableOrViewName
                 acquireLock(region, key, locks);
+                // Invalidate the cache from all the regionservers.
+                invalidateServerMetadataCache(tenantId, schemaName, tableOrViewName);
```

Comment:
We need to call invalidate cache in other scenarios also like create index. Will that come in a separate JIRA ?

Reply:
> We need to call invalidate cache in other scenarios also like create index. 

Yes, I have already created jira to update LAST_DDL_TIMESTAMP of the parent table/view when we add an index. [PHOENIX-7056](https://issues.apache.org/jira/browse/PHOENIX-7056)

Reply:
> Will that come in a separate JIRA ?

Yes.